src/EventTickets/Repositories/EventTicketTypeRepository.php

```diff
@@ -64,15 +60,16 @@ public function insert(EventTicketType $eventTicketType)
         try {
             DB::table('give_event_ticket_types')
                 ->insert([
-                    'id' => $eventTicketType->id,
                     'event_id' => $eventTicketType->event_id,
                     'label' => $eventTicketType->label,
                     'description' => $eventTicketType->description,
-                    'price' => $eventTicketType->price->formatToMinorAmount(),
+                    'price' => $eventTicketType->price->formatToDecimal(),
```

Comment:
Currency values should be stored as the minor amount of the currency.

Reply:
Reverted the change for now.